### PR TITLE
Scrolling Tabs

### LINKED ISSUE #40: Swipe to scroll doesn't work
Works beautifully in the 0.5 version...

### PR DESCRIPTION
**Pull request by azakus**
Fixes #40

### CODE REVIEW

**Comment by azakus** [comment]
Mouse is still a bit funky (ooops) due to a focus effect in IronMenuBehavior. Still working on the right way to disable it for tracking.

**Comment by cdata** [comment]
In trying this out locally, it appears as though ripple likes to be on the aggressive side:

paper-tabs

File: paper-tabs.html [hunk 1]
```diff
@@ -310,6 +310,10 @@
       'iron-deselect': '_onIronDeselect'
     },
 
+    ready: function() {
+      this.setScrollDirection('y', this.$.tabsContainer);
```
Comment (cdata) [inline paper-tabs.html]: What this do? :meat_on_bone:

File: paper-tabs.html [hunk 2]
```diff
@@ -364,24 +368,43 @@
...
         return;
       }
 
-      scrollLeft = this.$.tabsContainer.scrollLeft;
+      var ddx = (detail && -detail.ddx) || 0;
+      this._affectScroll(ddx);
+    },
+
+    _down: function(e) {
+      // go one beat async to defeat IronMenuBehavior
```
Comment (cdata) [inline paper-tabs.html]: :facepunch:

File: paper-tabs.html [hunk 3]
```diff
@@ -364,24 +368,43 @@
...
+      // autorefocus-on-no-selection timeout
+      this.async(function() {
+        if (this._defaultFocusAsync) {
```
Comment (cdata) [inline paper-tabs.html]: Do you need to assign this on line 384?

Comment (cdata) [inline paper-tabs.html]: (or.. is it assigned in `IronMenuBehavior`? If so, :facepunch: :facepunch: )

**Comment by cdata** [comment]
Minor questions, otherwise LGTM :+1:

File: paper-tabs.html (hunk 3 above)
Comment (azakus) [inline paper-tabs.html]: `_defaultFocusAsync` is used in IronMenuBehavior: https://github.com/PolymerElements/iron-menu-behavior/blob/master/iron-menu-behavior.html#L244-L255

File: paper-tabs.html (hunk 1 above)
Comment (azakus) [inline paper-tabs.html]: This turns off `track` if you try to drag in the vertical axis. I went over this slightly in my talk :no_mouth:

**Comment by cdata** [comment]
This LGTM :+1: thanks @azakus !

**Pull request merged**